.github/workflows/build.yml

```
@@ -18,7 +18,8 @@ jobs:
       - name: 'Dependencies'
         run: |
           apt-get update
-          apt-get install -y git python3.9 pip cmake ninja-build cudnn9-cuda-12
+          apt-get install -y git python3.8 pip ninja-build cudnn9-cuda-12
```

Python 3.8 doesn't seem to be available from `apt-get`. If we want to test compatibility with older Python versions, we'll probably need more dedicated logic.
```suggestion
          apt-get install -y git python3.9 pip ninja-build cudnn9-cuda-12
```